### ISSUE TITLE
Suggestion on the wiki doc

### ISSUE THREAD

**Issue by KinglittleQ** [facebookresearch/faiss#2269]
# Summary

The documentation of the index factory [here](https://github.com/facebookresearch/faiss/wiki/Faiss-indexes) is a little ambiguous and misleading. Maybe we could represent each factory string by f-string style.

For example:

```python

HNSW{M}_Flat

IVF{nlist},Flat

PQ{M}x{nbits}

```

**Comment by mdouze** [comment]
Yes right. In fact, this documentation is redundant with the index_facotry doc here: 
https://github.com/facebookresearch/faiss/wiki/The-index-factory
but your suggestion applies there as well.
